auth/sts.go

```diff
@@ -4,7 +4,6 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"github.com/mitchellh/go-homedir"
```

Comment:
`go-homedir` is removed in favor of the `ExpandTilde` function. I'm guessing that function is borrowed from the Internet? In other words, are we sure this works on all platforms?

Reply:
@bvdeenen see comment above